dev/NavigationView/NavigationView.cpp

```diff
@@ -3524,19 +3725,6 @@ void NavigationView::UpdateSelectedItem()
     {
```

Comment:
The naming seems incorrect. This is not updating anything. It is invoking the settings ?